lnst/Recipes/ENRT/GeneveLwtTunnelRecipe.py

```diff
@@ -215,8 +215,13 @@ def get_packet_assert_config(self, ping_config):
             pa_kwargs["p_filter"] = "ip6"
             grep_pattern = r"IP6 "
 
-        grep_pattern += r"{}\.[0-9]+ > {}\.[0-9]+: Geneve.*vni 0x4d2: ".format(
-            m1_carrier_ip, m2_carrier_ip
+        # 14:56:29.576709 IP 192.168.101.1.35551 > 192.168.101.2.6081: Geneve, Flags [none], vni 0x4d2, options [8 bytes]: IP 172.16.10.1 > 172.16.20.1: ICMP echo request, id 64, seq 1, length 64
+        options = r", options \[[0-9]+ bytes\]" if self.params.geneve_opts else ""
```

Comment:
is there a way to check the option value here as well or would that be better as a separate tcpdump check?

Reply:
At the moment this is not possible as the PacketAssert does not dump the detailed info. Maybe it would make sense to refactor it to work directly with a pcap file instead, I believe there's a library that could make parsing out the individual fields easier.

Reply:
ok